Replaces Rotciv with Colton in wizard academy

LoliconSlayer opened the pull request:
Self explanatory title

Oh yeah and adds Anvilman as a wizard because people kept complaining about Feweh not being a yogs reference, so instead of replacing Feweh with Colton I added another person because golly me do I fucking hate retards

:cl:  
tweak: Rotciv the Dented and Feweh the Replicator have been expelled from the wizard academy! In their place, Colton the Afterburner and Anvil the Archivist have transferred in.
/:cl:

LoliconSlayer committed:
I hate retards

Inb4 someone says "If you hate retards then does that mean you hate yourself?"

LoliconSlayer committed:
Stop complaining about other people

If you wanna complain about Feweh not being yogs related I'll fucking replace him too, I still don't wanna keep rotciv in because I hate retards

LoliconSlayer committed:
Anvilman6

Tessa got mad at me so Anvil time

patpol4 reviewed (dismissed):
Alexkar asked me to approve this as a test

LoliconSlayer committed:
Why did I even write scribes

Now that I think about it, several other archivists makes more sense than several other scribes, also it's archivist not archiver

patpol4 reviewed (changes requested):
Was going to close but this but im willing to go with Featherfields suggestion. Replace Rotciv with Colton, add a dented helmet with some stupid description. And remove Anvil, we don't add staff for merely existing.
